### PR TITLE
Text analysis reporting

### PR DESCRIPTION
**Pull request by boblannon**
Following up on #222, this script creates new database tables populated with unigram and bigram statistics.  It then produces meta-analysis tables suitable for reporting.

### CODE REVIEW

File: test/analyze.test.js [hunk 1]
```diff
@@ -32,41 +33,56 @@ test('Init db', (t) => {
...
+                (2, '{"wong ho ln"}', '{"wong ho"}', '{"Wong Ho Lane"}'),
+                (3, '{"pier 1"}', '{"pier 1"}', '{"Pier 1"}'),
+                (4, '{"main st"}', '{"main"}', '{"Main St"}'),
+                (5, '{"fake st"}', '{"fake"}', '{"Fake St"}'),
+                (6, '{"elm way"}', '{"elm"}', '{"Elm Way"}'),
+                (7, '{"evergreen tr"}', '{"evergreen"}', '{"Evergreen Terrace"}');
             COMMIT;
         `, (err, res) => {
-            t.error(err);
+            if (err) t.error(err);
```
Comment (ingalls) [inline test/analyze.test.js]: You should just be able to use `t.error` it checks the error falsyness itself.

Comment (boblannon) [reply test/analyze.test.js]: oh neat.  okay, will do.

**Comment by boblannon** [comment]
hey @ingalls can I get another look, now that tests are passing?

**Comment by ingalls** [comment]
🌮

**Pull request merged**